CONTRIBUTING.md

```diff
@@ -100,6 +100,41 @@ The packages are available at the following urls with hot-reloading:
 - Front-end: http://localhost:3000
 - Back-end: http://localhost:3100
 
+#### MongoDB Compass
+
+To access MongoDB with the [MongoDB Compass](https://www.mongodb.com/products/compass) GUI, you can do the following after opening MongoDB Compass:
+
+1. In the menu bar, click **Connect** and choose **New Connection**
+2. Paste the connection string you configured in your `.env.local` here
+3. Press **Connect**
+
+At this point you should be connected to the database and see your databases. Click into the desired database, e.g. growthbook, to view your collections.
```

Comment:
@Auz I've added the MongoDB Compass instructions here. This tool is great and much simpler (looks like the Atlas UI). This is definitely a more user-friendly option.